changelog/updates/2023-07-20-openldap-2.5.14.md

```diff
@@ -0,0 +1 @@
+- openldap ([2.5.14](https://lists.openldap.org/hyperkitty/list/openldap-announce@openldap.org/thread/TZQHR4SIWUA5BZTKDAKSFDOOGDVU4TU7/))
```

Comment:
Would it be a good idea to point to the 2.5 major version release announcement too?

```suggestion
- openldap ([2.5.14](https://lists.openldap.org/hyperkitty/list/openldap-announce@openldap.org/thread/TZQHR4SIWUA5BZTKDAKSFDOOGDVU4TU7/) (includes [2.5.4](https://lists.openldap.org/hyperkitty/list/openldap-announce@openldap.org/thread/BH3VDPG6IYYF5L5U6LZGHHKMJY5HFA3L/)))
```

Reply:
Either one is fine, 2.5.4 was the first version from 2.5 to be released for general use.